docs/audits/gpt-errors-overall.md

```diff
@@ -0,0 +1,16 @@
+# Minimize GPT API related errors
```

Comment:
Same as the other doc, does this exactly match the audit failure title?

Reply:
Done.